tests/bfd/test_bfd.py

```diff
@@ -12,6 +12,9 @@
 pytestmark = [
     pytest.mark.topology('t1', 't1-lag', 't1-64-lag')
 ]
+BFD_RESPONDER_SCRIPT_SRC_PATH = '../ansible/roles/test/files/helpers/bfd_responder.py'
+BFD_RESPONDER_SCRIPT_DEST_PATH = '/opt/bfd_responder.py'
+
 def skip_201911_and_older(duthost):
```

Comment:
```suggestion
```
This function can be removed, because of this PR #5708

Reply:
same as above